docs/builtin-functions.md

```diff
@@ -111,8 +111,8 @@ Python contains a limited number of builtin functions defined [here](https://doc
 | `remove` | Python-only |
 | `symmetric_difference` | No |
 | `symmetric_difference_update` | No |
-| `union` | No |
-| `update` | Python-only |
+| **`union`** | Yes |
+| **`update`** | Yes |
```

Comment:
is `update` still within the scope of this PR?

Reply:
Removed [24e886c](https://github.com/pyccel/pyccel/pull/2015/commits/24e886cccb8b6113bcfc92116b4203b363863d4a)